packages/flutter_tools/lib/src/base/utils.dart

```diff
@@ -27,6 +27,7 @@ class BotDetector {
 
         // Set by the IDEs to the IDE name, so a strong signal that this is not a bot.
         || platform.environment.containsKey('FLUTTER_HOST')
```

Comment:
nit: why not reusing `FLUTTER_HOST`. The comment above indicates that it's used by IDEs.

Reply:
Gonna let https://github.com/flutter/flutter/pull/36208 merge first, then this PR will just be tests